src/redux/missions/missionSlice.js

```diff
@@ -1,19 +1,34 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
 const initialState = {
   missions: [],
 };
 
+export const getMissions = createAsyncThunk(
+  'missions/getMissions',
+  () => axios.get('https://api.spacexdata.com/v3/missions'),
+);
+
 const missionsSlice = createSlice({
   name: 'missions',
   initialState,
   reducers: {
-    setMissions(state, { payload }) {
-      return {
+    setMissions: (state, { payload }) => ({ ...state, rockets: payload }),
```

Comment:
Hi @mrkamin, please, notice you are replacing `rockets` instead of `missions` array in your return statement.